datahub-frontend/app/controllers/AuthenticationController.java

```diff
@@ -168,4 +278,25 @@ private String encodeRedirectUri(final String redirectUri) {
             throw new RuntimeException(String.format("Failed to encode redirect URI %s", redirectUri), e);
         }
     }
+
+    private boolean tryLogin(boolean jaasEnabled, boolean nativeAuthenticationEnabled, String username, String password) {
```

Comment:
Why do we need to pass i these booleans? 

Don't we have access to config obj in this method?

Not blocking. Just curious in code style why we'd do this

Reply:
Fair point, refactored!